_posts/2023-09-22-OpenSearch-Personas-Creating-Figma-Templates-to-Represent-Person-Framework

```diff
@@ -0,0 +1,43 @@
+[Image: OpenSearch Journey Map Library.jpg]
+[Image: image]
+The sticker sheet provides an overview of the components as well as some variants. In addition, we provided a sample journey map so that you can obtain an idea of how to lay out your own journey map in accordance with your team’s requirements. 
+[Image: Sticker sheet 1.jpg][Image: image.png]Here is an example of a journey map template and one variation (dark mode). 
+[Image: image.png][Image: image.png]Here is how you create your own user journey map. On the left panel, you’ll see a tab called **Assets**. On this tab, you’ll see the components listed. There is a category for persona and a category for task flow. The persona category contains all of the components that are specific to personas. The task flow category contains everything needed to create a task flow diagram or a journey map. You can view these in a list view or a grid view. Hovering over a component provides information on how many variants there are. You can select this to see more information about the components. 
+
+First select a persona trait card. You will see various hierarchies of complexity on this card and will need user research data to populate it. Layers on the right corresponding to traits selected by the user can be toggled on or off. Additionally, you can select individual text fields, such as persona name, role, or description, and edit these based on user research findings. In addition to these components, you can edit nested components in order to incorporate complexities found in your user sample. 
+[Image: image.png]
+Now you can create journey maps incrementally based on the persona. The Figma template offers various components corresponding to the user’s phase, stage, what they aim to do, how they feel, how they think, the tools they use, pain points, opportunities, the level of effort needed to accomplish the phase, and the stakeholders involved at that point in time. The template is designed so that you can build the journey map step by step. You can duplicate each phase or create a specific component individually. Ultimately, once you have constructed each user phase, you can align various components as appropriate. Now you can select a frame and choose different themes (such as dark mode or color templates that you can predefine). The OpenSearch Project is in the process of creating core personas for search, observability, and security use cases. We will be using these templates to define users and journeys, and we hope you will find this useful in defining your users and experiences. 
+
+More OpenSearch Project resources available on Figma: https://www.figma.com/@opensearch
```

Comment:
add `<>` around all the links to make them clickable -- example:

`<https://www.figma.com/@opensearch>`